haystack/utils/deepsetcloud.py

```diff
@@ -1648,3 +1687,31 @@ def create_and_start_run(
             tags=tags,
         )
         cls.start_run(eval_run_name=eval_run_name, workspace=workspace, api_key=api_key, api_endpoint=api_endpoint)
+
+    @classmethod
+    def get_run_results(
```

Comment:
Let's use singular here as it will return one object.

```suggestion
    def get_run_result(
```